zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -56,6 +57,12 @@
 }
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
```

Comment:
Can you remove this and use `XiaomiAqaraE1Cluster` directly in the `replacement` below?

Reply:
done